llvm/test/DebugInfo/extradata-node-reference.ll

```diff
@@ -0,0 +1,91 @@
+;; Test verifies that node reference in the extraData field are handled correctly
+;; when used with tags like DW_TAG_member, DW_TAG_inheritance etc.
+
+; REQUIRES: object-emission
+; RUN: %llc_dwarf %s -filetype=obj -o - | llvm-dwarfdump - | FileCheck %s
```

Comment:
Could we get a bitcode round-trip check too please? (`opt -S  | llvm-as | llvm-dis`, FileCheck directives to check the metadata refs still look right)